### PR TITLE
Feat : Images

### PR DESCRIPTION
**Pull request by thesudhanshupanda**

### CODE REVIEW

File: docs/Images/Image.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                <li>
+                    <a href="https://github.com/thesudhansupanda/Merakii-UI">Github</a>
+                </li>
+            </ul>
+        </nav>
+        <div class="menu-div">
+            <button class="btn btn-primary btn-float" id="menu-btn">
+              <i class="fas fa-bars"></i>
+            </button>
+          </div>
```
Comment (shuntzhere) [inline docs/Images/Image.html:29]: Check this. Are you deliberately trying to put just one button in this menu div?

File: docs/Images/Image.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                            <div class="component-content">
+                                <div class="component-heading">Round and Square Image</div>
+                                <div class="component-subHeading">  
+                                Images with Default dimension of 200px X 200px.            
+                                </div>
+                                <!-- Round Image-->
+                                <div class="example-live flex-wrap">                                
+                                   <img class="img-md round-img" src="/Assets/spacexnasa.png" alt="">
+                                   <img  class="img-md square-img" src="/Assets/spacexnasa.png" alt="">                               
+                                </div>
```
Comment (shuntzhere) [inline docs/Images/Image.html:77]: Give an alt text.

**Review by shuntzhere — COMMENTED** [review]
Classes are well-named. Do indent the code once.
Also suggested a few changes do give them a look.

File: docs/Images/Image.html (hunk 2 above)
Comment (thesudhanshupanda) [reply docs/Images/Image.html:77]: alright thanks

**Review by shuntzhere — APPROVED** [review]
Looks good, do add some description with image or video for better understanding.

File: docs/Images/Image.html (hunk 1 above)
Comment (thesudhanshupanda) [reply docs/Images/Image.html:29]: yes , its for mobile view !

File: docs/Images/Image.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                                 </div>
+                                    <div class="example-live ">
+                                    <div class="img-wrapper">
+                                        <img class="resp-img" src="/Assets/spacexnasa.png">
+                                    </div> 
+                                    </div>
+                                      <div class="example-code">
+                                         <pre>
+                                              <code class="html hljs xml">
+                                               <img class="resp-img" src="Paste image url" alt="">
```
Comment (krishanu2911) [inline docs/Images/Image.html:101]: do not leave the alt tag empty

File: docs/Images/Image.html [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+                            <!-- Components -->
+                            <div class="component-content">
+                                <div class="component-heading">Round and Square Image</div>
+                                <div class="component-subHeading">  
+                                Images with Default dimension of 200px X 200px.            
+                                </div>
+                                <!-- Round Image-->
+                                <div class="example-live flex-wrap">                                
+                                   <img class="img-md round-img" src="/Assets/spacexnasa.png" alt="">
+                                   <img  class="img-md square-img" src="/Assets/spacexnasa.png" alt="">
```
Comment (krishanu2911) [inline docs/Images/Image.html:76]: same here

File: docs/Images/Image.html [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+                            </div>  
+                            <!-- Components -->
+                            <div class="component-content">
+                                <div class="component-heading">Round and Square Image</div>
+                                <div class="component-subHeading">  
+                                Images with Default dimension of 200px X 200px.            
+                                </div>
+                                <!-- Round Image-->
+                                <div class="example-live flex-wrap">                                
+                                   <img class="img-md round-img" src="/Assets/spacexnasa.png" alt="">
```
Comment (krishanu2911) [inline docs/Images/Image.html:75]: same here

**Review by krishanu2911 — APPROVED** [review]
Everything is good to go no major changes are required just added a few corrections.

File: docs/Images/Image.html (hunk 3 above)
Comment (thesudhanshupanda) [reply docs/Images/Image.html:101]: alright thanks

**Pull request merged**